### PR TITLE
All events functionality

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #106

### CODE REVIEW

File: src/actions/get_all_events.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+const getAllEvents = (hbs, response) => {
+  mockEvents.find({}, (error, events) => {
+    if (error) response.render('not_found')
+    else {
+      let upcomingEvents = [];
+      let pastEvents = [];
+      events.forEach((event) => {
+        if(event.eventDate > new Date()) {
```
Comment (esraajb) [inline src/actions/get_all_events.js]: https://www.w3schools.com/js/tryit.asp?filename=tryjs_date_new
this is what new Date returns, this is how you wanna do it?

File: tests/server/index.js [hunk 2]
```diff
@@ -122,14 +122,25 @@ tape('Test for results', (t) => {
   })
 })
```
Comment (esraajb) [inline tests/server/index.js]: these are very basic tests, you should think of new ways to do testing

File: src/actions/get_all_events.js (hunk 1 above)
Comment (Suuuuha) [inline src/actions/get_all_events.js]: Javascript has no problem with dates in if statement and its working, and I think event.eventDate returns the same date eyntax.
And for the date format we have a helper function to change it to String when its rendered to the page.

File: tests/server/index.js (hunk 2 above)
Comment (Suuuuha) [inline tests/server/index.js]: @esraajb do you mean that you prefer that we test each function seperately?

**Comment by esraajb** [comment]
@Suuuuha merge conflict

**Pull request merged**